### ISSUE TITLE
Failed to validate GPG signature for amazon-ssm-agent-*

### ISSUE THREAD

**Issue by eexit** [dhoeric/ansible-aws-ssm#16]
Hello,

I was trying this role on Centos 8 then I got this error:

```
amazon-ebs: TASK [amazon-ssm-agent : Get CPU architecture] *********************************
    amazon-ebs: ok: [127.0.0.1] => {"changed": false, "cmd": ["getconf", "LONG_BIT"], "delta": "0:00:00.006878", "end": "2021-03-29 18:46:09.663904", "rc": 0, "start": "2021-03-29 18:46:09.657026", "stderr": "", "stderr_lines": [], "stdout": "64", "stdout_lines": ["64"]}
    amazon-ebs:
    amazon-ebs: TASK [amazon-ssm-agent : Change URL destination for 32bit arch] ****************
    amazon-ebs: skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}
    amazon-ebs:
    amazon-ebs: TASK [amazon-ssm-agent : Install rpm file for Redhat Family (Amazon Linux, RHEL, and CentOS) 32/64-bit] ***
    amazon-ebs: fatal: [127.0.0.1]: FAILED! => {"changed": false, "msg": "Failed to validate GPG signature for amazon-ssm-agent-3.0.882.0-1.x86_64"}
```

It seems related to [this issue](https://github.com/aws/amazon-ssm-agent/issues/235), which has been "resolved" but I still think the verification process is cumbersome. Any idea how to implement this process as part of this role?

Thanks!

**Comment by dhoeric** [comment]
Hi @eexit, the issue is fixed and released in v1.0.2. You can download via 

```shell
ansible-galaxy install dhoeric.aws-ssm,v1.0.2
```

**Comment by eexit** [comment]
Thanks!!!
